### PR TITLE
 [RTC-216/217] Allow token to be set in config and update docs

### PR DESCRIPTION
**Pull request by LVala**
This PR contains [RTC-126](https://membraneframework.atlassian.net/jira/software/c/projects/RTC/boards/18?modal=detail&selectedIssue=RTC-216) and [RTC-217](https://membraneframework.atlassian.net/jira/software/c/projects/RTC/boards/18?modal=detail&selectedIssue=RTC-217) as they are closely related.

[RTC-126]: https://membraneframework.atlassian.net/browse/RTC-126?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RTC-217]: https://membraneframework.atlassian.net/browse/RTC-217?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/jellyfish/client.ex [hunk 1]
```diff
@@ -29,14 +29,34 @@ defmodule Jellyfish.Client do
...
     * `address` - url or IP address of the Jellyfish server instance
     * `token` - token used for authorizing HTTP requests. It's the same
-    token as the one configured in Jellyfish.
+    token as the one configured in Jellyfish. If not passed, value set via
+    `config :jellyfish_server_sdk, token: "your-token"` in `config.exs` is used.
   """
-  @spec new(String.t(), String.t()) :: t()
-  def new(address, token) do
+  @spec new(String.t(), String.t() | nil) :: t()
+  def new(address, token \\ nil)
```
Comment (mickel8) [inline lib/jellyfish/client.ex:50]: That's a nitpick but I think I would go for `new/1` and `new/2`

**Review by mickel8 — APPROVED** [review]

File: lib/jellyfish/client.ex (hunk 1 above)
Comment (LVala) [reply lib/jellyfish/client.ex:50]: I'll do that, it seems like an obvious choice, I don't know why I went for the default argument.

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ room_id
 # => "8878cd13-99a6-40d6-8d7e-8da23d803dab"
 
 # Add peer
-{:ok, %Jellyfish.Peer{id: peer_id}, token} = Jellyfish.Room.add_peer(client, room_id, "webrtc")
+{:ok, %Jellyfish.Peer{id: peer_id}, client_token} = Jellyfish.Room.add_peer(client, room_id, "webrtc")
```
Comment (mickel8) [inline README.md:39]:
```suggestion
{:ok, %Jellyfish.Peer{id: peer_id}, peer_token} = Jellyfish.Room.add_peer(client, room_id, "webrtc")
```

File: lib/jellyfish/client.ex [hunk 3]
```diff
@@ -36,7 +36,28 @@ defmodule Jellyfish.Client do
     token as the one configured in Jellyfish.
   """
   @spec new(String.t(), String.t()) :: t()
-  def new(address, token) do
+  def new(address, token), do: build_client(address, token)
+
+  @doc """
+  Creates new instance of `t:Jellyfish.Client.t/0`.
```
Comment (mickel8) [inline lib/jellyfish/client.ex:42]: JFYI, you can also write `t:t/0` but the result will be slighlity different

File: lib/jellyfish/room.ex [hunk 4]
```diff
@@ -40,6 +40,11 @@ defmodule Jellyfish.Room do
   """
   @type id :: String.t()
 
+  @typedoc """
+  Client token, created by Jellyfish. Required by client application to open connection to Jellyfish.
+  """
+  @type client_token :: String.t()
```
Comment (mickel8) [inline lib/jellyfish/room.ex:46]: Could we name this peer_token?

**Review by mickel8 — CHANGES_REQUESTED** [review]

**Review by mickel8 — APPROVED** [review]

**Pull request merged**